spec/core/data_structures.md

```diff
@@ -270,8 +270,8 @@ The vote includes information about the validator signing it. When stored in the
 | Name               | Type                            | Description                                                                              | Validation                               |
 |--------------------|---------------------------------|------------------------------------------------------------------------------------------|------------------------------------------|
 | Type               | [SignedMsgType](#signedmsgtype) | The type of message the vote refers to                                                   | Must be `PrevoteType` or `PrecommitType` |
-| Height             | int64                           | Height for which this vote was created for                                               | Must be > 0                              |
-| Round              | int32                           | Round that the commit corresponds to.                                                    | Must be > 0                              |
+| Height             | int64                           | Height for which this vote was created for                                               | Must be >= 0                             |
```

Comment:
When is this field set to 0 ?
Looks like it can't be zero, because of [this](https://github.com/cometbft/cometbft/blob/main/types/vote.go#L279)